src/serializer/MapFishPrintV3OSMSerializer.js

```diff
@@ -52,12 +52,13 @@ export class MapFishPrintV3OSMSerializer extends BaseSerializer {
       imageExtension: 'png',
       maxExtent: [-20037508.34, -20037508.34, 20037508.34, 20037508.34],
       rasterStyle: undefined,
-      resolutionTolerance: 0,
       resolutions: [],
       tileSize: [256, 256]
     });
 
     const source = layer.getSource();
+    const tileGrid = source.getTileGrid();
```

Comment:
Is the tilegrid always defined?

Reply:
If an OL OSM source is defined, then this should always be defined, but I can add a safity measure

Reply:
Ok, thanks for clarifying!